assets/i18n/strings/compile.js

```diff
@@ -36,7 +36,14 @@ async function main() {
       if (!localesPerMessage[name]) {
         localesPerMessage[name] = {};
       }
-      localesPerMessage[name][locale] = $msg.html();
+      localesPerMessage[name][locale] = $msg
+        .html()
+        // Simplify template.
+        .replace('<strong></strong>', '')
```

Comment:
Aren't the strings exported from our own tools? Can't we update our exporter to not include the tag if it isn't necessary? I'd prefer that over updating our compiler for things like this

Reply:
Will do!